spec/requests/api/v1/users_request_spec.rb

```diff
@@ -149,4 +149,27 @@
       end
     end
   end
+
+  describe 'deletes a user' do
+    describe 'happy path' do
+      it "finds a user by id and deletes them" do
+        user_1 = create(:user, id: 1)
+
+        delete api_v1_user_path(1)
+
+        expect(response.status).to eq(204)
+        expect(response.body).to be_empty
+      end
+    end
+
+    describe 'sad path' do
+      it 'cannot find the record to delete and returns an error' do
+        user_1 = create(:user, id: 1)
+
+        delete api_v1_user_path(99999)
+
+        expect(response.status).to eq(204)
```

Comment:
So it should be getting a 404 back not 204 right?

Reply:
Mal said she fixed that on a different branch she will merge later, but it has been taken care of.